### PR TITLE
update 'generate_ipfs_hashes' script

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

The update was needed because of the recent upgrade of the package `ipfshttpclient`. With the new version, the flag that turns on the recursive computation of hashes of a directory was set to `False` by default, which wasn't the desired outcome.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments
n/a

### CODE REVIEW

File: tests/data/hashes.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-dummy_author/agents/dummy_aea,QmR4SuPKbe4qjgF5DVwVwHPm5eFqCx83sdavs4QEXgF1mR
+dummy_author/agents/dummy_aea,QmVjFRYLEwTF9zK2g4sDTvb3Djo2We1QFiNUHT6EJedgsx
```
Comment (marcofavorito) [inline tests/data/hashes.csv:1]: this is likely due to symlinks. `follow_symlinks=True` didn't solve the difference.

File: tox.ini [hunk 2]
```diff
@@ -74,7 +74,7 @@ commands = {toxinidir}/scripts/check_copyright_notice.py --directory {toxinidir}
 [testenv:hash_check]
 skipsdist = True
 usedevelop = True
-deps = ipfshttpclient
+deps = ipfshttpclient==0.6.0.post1
```
Comment (marcofavorito) [inline tox.ini:77]: better to stick with one version to avoid surprises.

Comment (DavidMinarsch) [reply tox.ini:77]: Needs to be properly bumped in pipenv too

File: tests/data/hashes.csv (hunk 1 above)
Comment (DavidMinarsch) [reply tests/data/hashes.csv:1]: Concerning. We need to understand why before we merge this.

**Comment by DavidMinarsch** [comment]
Closed in favour of #1588

**Pull request closed without merging**